hexrd/ui/image_load_manager.py

```diff
@@ -319,33 +323,23 @@ def add_omega_metadata(self, ims_dict):
         # Add on the omega metadata if there is any
         files = self.data['yml_files'] if 'yml_files' in self.data else self.files
         for key in ims_dict.keys():
+            nframes = self.data['total_frames'][0] * len(files[0])
```

Comment:
Do we have the raw imageseries loaded at this point?  If so, `nframes = len(raw_ims)`